std_msgs/Header header
bool do_calib
bool ack_calib